src/NHibernate.Test/NHSpecificTest/NH1001/Fixture.cs

```diff
@@ -158,5 +179,26 @@ public void AddressNull()
 				Assert.That(statistics.PrepareStatementCount, Is.EqualTo(1));
 			}
 		}
+
+		[Test]
+		public void PhoneIsNull()
+		{
+			ExecuteStatement($"UPDATE EMPLOYEES SET DEPARTMENT_ID_1 = 11, DEPARTMENT_ID_2 = 12, ADDRESS_ID = 15 WHERE EMPLOYEE_ID = {employeeId}");
+			ExecuteStatement($"UPDATE PHONES SET EMPLOYEE_ID = NULL");
+
+			IStatistics statistics = Sfi.Statistics;
+			statistics.Clear();
+
+			using (ISession session = OpenSession())
+			{
+				var employee = session.Get<Employee>(employeeId);
+				Assert.That(employee.Department1, Is.Not.Null);
+				Assert.That(employee.Department2, Is.Not.Null);
+				Assert.That(employee.Department3, Is.Not.Null);
+				Assert.That(employee.Address, Is.Not.Null);
+				Assert.That(employee.Phones, Is.Null);
```

Comment:
I'm afraid but I don't think this is correct behavior. Why would you want you `Phones` collection be initialized with null? It should be simply empty collection. And I believe it's the way it's already working without any fixes.

As I already said I don't understand at all what's use for [`not-found`  in one-to-many](https://nhibernate.info/doc/nhibernate-reference/collections.html#collections-onetomany) association
@hazzik , @fredericDelaporte  Can you shed some light on it?